Source/WebKit/UIProcess/API/Cocoa/WKPreferences.mm

```diff
@@ -1713,6 +1713,24 @@ - (BOOL)_spatialVideoEnabled
 #endif
 }
 
+- (void)_setWebPageSpatialBackdropEnabled:(BOOL)enabled
+{
+#if ENABLE(WEB_PAGE_SPATIAL_BACKDROP)
+    _preferences->setWebPageSpatialBackdropEnabled(enabled);
+#else
+    UNUSED_PARAM(enabled);
+#endif
+}
+
+- (BOOL)_webPageSpatialBackdropEnabled
+{
+#if ENABLE(WEB_PAGE_SPATIAL_BACKDROP)
+    return _preferences->webPageSpatialBackdropEnabled();
+#else
+    return NO;
+#endif
+}
```

Comment:
I kinda doubt we want to modify this file? As I understand it this is a legacy file from before UniviedWebPreferences.yaml.

Reply:
That's fair, but I think new things only get added when needed. Will there be a caller for this?